src/Umbraco.Tests.Common/Builders/Extensions/ContentTypeBuilderExtensions.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Umbraco.Core;
+using Umbraco.Core.Models;
+
+namespace Umbraco.Tests.Common.Builders.Extensions
+{
+    public static class ContentTypeBuilderExtensions
+    {
+        public static ContentType BuildSimpleContentType(this ContentTypeBuilder builder)
```

Comment:
This method should in theory be closer to what a `new ContentTypeBuilder().Build()` should do.

I think we can use more default value in the builders: E.g.
- icon = `icon-document`
- thumbnail = `folder.png`
- path = `_path ?? $"-1,{GetId()}"`

Reply:
In general the reason I've done this is just to make sure I'm not changing what the test was previously doing - i.e. if it was setting a value before, I've left it doing it still.  But I do agree default values would serve the purpose of the tests in most cases just as well.  I'll have a look to where we can use these more.

Reply:
Have made some updates to address this.